proto-lens-protoc/src/Data/ProtoLens/Compiler/Combinators.hs

```diff
@@ -67,30 +67,41 @@ recDecl dataName fields
 type Decl = Syntax.Decl ()
 
 patSynSig :: Name -> Type -> Decl
+#if MIN_VERSION_haskell_src_exts(1,20,0)
```

Comment:
I'm curious if anybody ran analysis on all Hackage for MIN_VERSION CPP gunk, which package would come out on top?

Reply:
In all likelihood, probably the `base` package itself :p
https://www.google.com/search?ei=7IleWufIDOPU0gLbj7Ug&q=%22min_version_base%22+site%3Ahackage.haskell.org&oq=%22min_version_base%22+site%3Ahackage.haskell.org&gs_l=psy-ab.3...6405.9216.0.9445.7.6.0.0.0.0.106.549.4j2.6.0....0...1c.1.64.psy-ab..2.0.0....0.x18W9bQot7A